docs/how-to-guides/upgrade-rook-ceph.md

```diff
@@ -0,0 +1,180 @@
+# Upgrading Rook Ceph
+
+## Contents
+
+- [Introduction](#introduction)
+- [Steps](#steps)
+  - [Step 1: Ensure `AUTOSCALE` is `on`](#step-1-ensure-autoscale-is-on)
+  - [Step 2: Watch](#step-2-watch)
+    - [Step 2.1: Ceph status](#step-21-ceph-status)
+    - [Step 2.2: Pods in rook namespace](#step-22-pods-in-rook-namespace)
+    - [Step 2.3: Rook version update](#step-23-rook-version-update)
+    - [Step 2.4: Ceph version update](#step-24-ceph-version-update)
+    - [Step 2.5: Events in rook namespace](#step-25-events-in-rook-namespace)
+  - [Step 3: Dashboards](#step-3-dashboards)
+    - [Step 3.1: Ceph](#step-31-ceph)
+    - [Step 3.2: Grafana](#step-32-grafana)
+  - [Step 4: Make a note of existing image versions](#step-4-make-a-note-of-existing-image-versions)
+  - [Step 5: Perform updates](#step-5-perform-updates)
+  - [Step 6: Verify that the CSI images are updated](#step-6-verify-that-the-csi-images-are-updated)
+  - [Step 7: Final checks](#step-7-final-checks)
+- [Additional resources](#additional-resources)
+
+## Introduction
+
+Rook Ceph is one of the storage providers of Lokomotive. With a distributed system as complex as
+Ceph, the upgrade process is not trivial. This document enlists steps on how to perform the upgrade
+and how to monitor this process.
+
+## Steps
+
+Following steps are inspired by [`rook`](https://rook.io/docs/rook/master/ceph-upgrade.html) docs.
+
+### Step 1: Ensure `AUTOSCALE` is `on`
```

Comment:
```suggestion
### Step 1: Ensure that all pools have the `AUTOSCALE` plugin enabled
```

Reply:
It can be enabled without being `on` so I have rephrased it.